Sources/Contentful/Resource.swift

```diff
@@ -236,36 +243,36 @@ public extension Dictionary where Key: ExpressibleByStringLiteral {
      - Parameter key: The name of the field to extract the `Bool` value from.
      - Returns: The `Bool` value, or `nil` if data contained is not convertible to a `Bool`.
      */
-    public func location(at key: Key) -> CLLocationCoordinate2D? {
+    public func location(at key: Key) -> Location? {
         let coordinateJSON = self[key] as? [String: Any]
         guard let longitude = coordinateJSON?["lon"] as? Double else { return nil }
         guard let latitude = coordinateJSON?["lat"] as? Double else { return nil }
-        let location = CLLocationCoordinate2D(latitude: latitude, longitude: longitude)
+        let location = Location(latitude: latitude, longitude: longitude)
         return location
     }
 
 }
 
 // MARK: Internal
 
-extension Resource: Hashable {
+extension LocalizableResource: Hashable {
     public var hashValue: Int {
         return id.hashValue
     }
 }
 
-extension Resource: Equatable {}
-public func == (lhs: Resource, rhs: Resource) -> Bool {
+extension LocalizableResource: Equatable {}
+public func == (lhs: LocalizableResource, rhs: LocalizableResource) -> Bool {
     return lhs.id == rhs.id && lhs.sys.updatedAt == rhs.sys.updatedAt
 }
 
 
-internal func +=<K: Hashable, V> (left: [K: V], right: [K: V]) -> [K: V] {
+internal func +=<K, V> (left: [K: V], right: [K: V]) -> [K: V] {
     var result = left
     right.forEach { (key, value) in result[key] = value }
     return result
 }
 
-internal func +<K: Hashable, V> (left: [K: V], right: [K: V]) -> [K: V] {
+internal func +<K, V> (left: [K: V], right: [K: V]) -> [K: V] {
```

Comment:
I fear this might not the right implementation of `+`. Also I feel I stumbled upon this before: Is left passed in as a reference or as a copy?

If it is a copy (aka changing it in here, does not change the variable outside) then I take back my comment and bow over in front of swift.

Reply:
it is a copy in swift since dictionaries are structs. when you pass one in, it is copied. we've had this conversation before ;-)